sljit_src/sljitNativeX86_common.c

```diff
@@ -3117,15 +3121,19 @@ SLJIT_API_FUNC_ATTRIBUTE struct sljit_put_label* sljit_emit_put_label(struct slj
 SLJIT_API_FUNC_ATTRIBUTE void sljit_set_jump_addr(sljit_uw addr, sljit_uw new_target, sljit_sw executable_offset)
 {
 	SLJIT_UNUSED_ARG(executable_offset);
+	SLJIT_UPDATE_WX_FLAGS((void*)addr, (void*)addr + (sizeof(sljit_uw)), 0);
```

Comment:
for portability (void *) arithmetic should be avoided, probably use sljit_uw for the cast + 1?

same below

Reply:
This is an unaligned address, and I know the compiler will not complain, but C expects us to use aligned pointers, so I choose this form since it represents a range of data rather than a value.

Reply:
then better use (char *) or its sljit equivalent (maybe sljit_u8 *), the problem I have is that sizeof(void *) != 1